commands/slashcommands.go

```diff
@@ -164,13 +164,23 @@ func (p *Plugin) yagCommandToSlashCommand(cmd *dcmd.RegisteredCommand) *discordg
 	}
 	t := true
 
+	integrationtypes := []int{0}
+	contexts := []int{0}
+
+	if cast.RunInDM {
+		integrationtypes = append(integrationtypes, 1) // USER_INSTALL
+		contexts = append(contexts, 1, 2)              // BOT_DM, PRIVATE_CHANNEL
```

Comment:
Can we have newtypes `IntegrationType`, `InteractionContextType` or similar for these?

Reply:
Like this?
![image](https://github.com/botlabs-gg/yagpdb/assets/65233836/ef7f4dd4-bb1f-4076-88e5-4a686465b51c)

Reply:
No, I meant constants replacing the magic numbers `1`, `2`, etc. See, e.g. `ForumLayout`:
https://github.com/botlabs-gg/yagpdb/blob/447a52732f5b901314397fe9079290e59d9f8d45/lib/discordgo/structs.go#L172-L181

Reply:
Ah that makes more sense, do you think I should still have the IntegrationTypes and IntegrationContexts types though?

Reply:
No, `[]IntegrationType` and `[]IntegrationContext` is clear enough IMO.

Reply:
Agreed

Reply:
Have a const for authorizing_integration_owners too, correct?

Reply:
Actually, I'm not entirely sure how I could do that for the "1" key because the value is a userID.

Reply:
Sorry, I'm not quite sure what the issue is. Could you elaborate?

Reply:
Nevermind, I was overthinking it lol